gateway-provider-security-jwt/src/main/java/org/apache/knox/gateway/provider/federation/jwt/JWTMessages.java

```diff
@@ -69,4 +69,8 @@
             text = "The configuration value ({0}) for maximum token verification cache is invalid; Using the default value." )
   void invalidVerificationCacheMaxConfiguration(String value);
 
+  @Message( level = MessageLevel.ERROR,
```

Comment:
The context from which this is logged is not applicable to the JWT scenario. It is only ever logged when the token ID is used as a passcode, which requires server-managed token state to be enabled because we can't authenticate based on the ID alone (we need the expiry/metadata). In this context, I believe this is an error.

Reply:
okay.